simtfl/util.py

```diff
@@ -8,3 +8,8 @@ def skip():
     """
     # Make this a generator.
     yield from []
+
+
+class Unique:
```

Comment:
This appears to be unused code. Generally I prefer each PR to contain only used code to keep the overall code as lean as possible. This is because I believe "anticipatory code" often needs to be modified once used, and worse it can sometimes accrue while remaining unused unintentionally, leading to unnecessary complexity.

Reply:
It's not unused; the `hash` value of a `BCBlock` is set to a `Unique` value. Again, I know this will be used, and being careful to make sure that the boundaries of PRs are such that everything is used in each PR is IMHO just make-work.

(If we were programming a production system then I'd agree with you; for prototyping, I don't. We're already doing this *way* more formally than is typical for prototyping.)

Reply:
It is nontrivially used now.